.gitlab-ci.yml

```diff
@@ -157,6 +157,32 @@ parthenon-cuda-performance_and_regression-mpi:
       - build-cuda-perf-mpi/tst/regression/outputs/advection_convergence_mpi/advection-errors.dat
       - build-cuda-perf-mpi/tst/regression/outputs/advection_convergence_mpi/advection-errors.png
 
+parthenon-cuda-mpi-examples-library:
+  only:
+    - schedules
+    - develop
```

Comment:
What's the develop run condition do?

Reply:
the `only` keyword takes the following names as logical or. So this ci workflow will run when the `develop` branch is triggered or when a `scheduled` test run is triggered etc.